dev/local-format.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+
+const { generateLinks } = require('../.github/workflows/format-release-note');
+
+const getMarkdownFiles = (dir) => {
+  let results = [];
+  const list = fs.readdirSync(dir);
+  list.forEach(file => {
+    file = path.resolve(dir, file);
+    const stat = fs.statSync(file);
+    if (stat && stat.isDirectory()) {
+      results = results.concat(getMarkdownFiles(file));
+    } else {
+      if (file.endsWith("release.md") && !isPreRelease(file)) {
```

Comment:
after changing to `{ withFileTypes: true }`, you can get the filename with `file.name`

```suggestion
      if (file.name.endsWith("release.md") && !isPreRelease(file.name)) {
```